cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinder.java

```diff
@@ -329,70 +330,126 @@ public void configure(Map<String, ?> configs) {
     }
     // Config for slow broker removal.
     Map<String, Object> originalConfig = _kafkaCruiseControl.config().originals();
-    _slowBrokerRemovalEnabled = Boolean.parseBoolean((String) originalConfig.get(SELF_HEALING_SLOW_BROKERS_REMOVAL_ENABLED_CONFIG));
-    try {
-      _bytesInRateDetectionThreshold = Double.parseDouble((String) originalConfig.get(SLOW_BROKERS_BYTES_IN_RATE_DETECTION_THRESHOLD_CONFIG));
-      if (_bytesInRateDetectionThreshold < 0) {
-        throw new IllegalArgumentException(String.format("%s config of slow broker finder should not be set to negative.",
-                                                         SLOW_BROKERS_BYTES_IN_RATE_DETECTION_THRESHOLD_CONFIG));
+    _slowBrokerRemovalEnabled = Boolean.parseBoolean((String) originalConfig.get(SELF_HEALING_SLOW_BROKER_REMOVAL_ENABLED_CONFIG));
+
+    String bytesInRateDetectionThreshold = (String) originalConfig.get(SLOW_BROKER_BYTES_IN_RATE_DETECTION_THRESHOLD_CONFIG);
+    if (bytesInRateDetectionThreshold == null) {
+      _bytesInRateDetectionThreshold = DEFAULT_SLOW_BROKER_BYTES_IN_RATE_DETECTION_THRESHOLD;
+    } else {
+      try {
+        _bytesInRateDetectionThreshold = Double.parseDouble(bytesInRateDetectionThreshold);
+        if (_bytesInRateDetectionThreshold < 0) {
+          throw new ConfigException(String.format("%s config of slow broker finder should not be set to negative, provided: %f.",
+                                                  SLOW_BROKER_BYTES_IN_RATE_DETECTION_THRESHOLD_CONFIG, _bytesInRateDetectionThreshold));
+        }
+      } catch (NumberFormatException e) {
+        throw new ConfigException(String.format("%s config of slow broker finder is invalid, provided: %s.",
```

Comment:
Applies to the other `NumberFormatException` -> can we indicate why it is invalid?